File: CursoPython3/exer/lista_Algoritimos/algoritimo/exer07.py
"""
7. Entrar com o dia e o mês de uma data e informar quantos dias se passaram
desde o início do ano. Esqueça a questão dos anos bissextos
e considere sempre que um mês possui 30 dias.
"""

day = int(input("Digite o dia: "))
month = int(input("Digite o mês: "))

days_passed = (month - 1) * 30 + day

print(f"Já se passaram {days_passed} dias desde o início do ano.")

File: CursoJava/Exer/Algoritimo/src/Exer07.java
/*
7. Entrar com o dia e o mês de uma data e informar quantos dias se passaram desde o início do
ano. Esqueça a questão dos anos bissextos e considere sempre que um mês possui 30 dias.
 */

import java.util.Scanner;

public class Exer07 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Digite o dia: ");
        int dia = sc.nextInt();

        System.out.println("Digite o mês: ");
        int mes = sc.nextInt();

        int dias = (mes - 1) * 30 + dia;

        System.out.println("Já se passaram " + dias + " dias desde o início do ano");

        sc.close();
    }
}
